rwengine/src/objects/GameObject.hpp

```diff
@@ -92,16 +87,30 @@ class GameObject {
         return static_cast<T*>(modelinfo_);
     }
 
-    /**
-     * @return The model used in rendering
-     */
-    ClumpPtr getModel() const {
+    const Model& getModel() const {
         return model_;
     }
 
-    /**
-     * Changes the current model, used for re-dressing chars
-     */
+     const AtomicPtr& getAtomic() const {
+        if (auto atomic = std::get_if<AtomicPtr>(&model_))
+        {
+            return *atomic;
+        }
+        return NullAtomic;
+    }
+
+    const ClumpPtr& getClump() const {
+        if (auto clump = std::get_if<ClumpPtr>(&model_))
+        {
+            return *clump;
+        }
+        return NullClump;
```

Comment:
Maybe it'd better insert static instance of object into body of function? (above return)

Reply:
Putting a local static here will generate some guard instructions, unlike a static member. [example](https://godbolt.org/z/M3K8RB).